pkg/cli/admin/prune/deployments/data.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"time"
 
+	k8sappsv1 "k8s.io/api/apps/v1"
```

Comment:
Nit: `kappsv1` is the preferred alias for k8s-based aliases that will collide with ours.